test/cri-containerd/removepodsandbox_test.go

```diff
@@ -0,0 +1,156 @@
+// +build functional
+
+package cri_containerd
+
+import (
+	"context"
+	"testing"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	runtime "k8s.io/cri-api/pkg/apis/runtime/v1alpha2"
+)
+
+type TestConfig struct {
+	name             string
+	containerName    string
+	requiredFeatures []string
+	runtimeHandler   string
+	sandboxImage     string
+	containerImage   string
+	cmd              []string
+}
+
+// Utility function to test removal of a sandbox with no containers and no previous call to stop the pod
+func runPodSandboxTestWithoutPodStop(t *testing.T, request *runtime.RunPodSandboxRequest) {
+	client := newTestRuntimeClient(t)
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	podId := runPodSandbox(t, client, ctx, request)
+	removePodSandbox(t, client, ctx, podId)
+
+	_, err := client.PodSandboxStatus(ctx, &runtime.PodSandboxStatusRequest{
+		PodSandboxId: podId,
+	})
+
+	if status, ok := status.FromError(err); !ok || status.Code() != codes.NotFound {
```

Comment:
Maybe just a nit, but I would prefer this line to be broken up like:

```
status, ok := status.FromError(err)
if !ok || status.Code() != codes.NotFound 
```

But again that's a nit so I'm fine with this being ignored :p

Reply:
sure!